src/views/Orders.vue

```diff
@@ -392,6 +396,36 @@ export default defineComponent({
         part.items.map((item: any) => items.push(item))
         return items;
       }, [])
+    },
+    async sendReadyForPickupEmail(order: any) {
+      const header = this.$t('Resend ready for pickup email')
+      const message = this.$t('An email notification will be sent to that their order is ready for pickup.', { customerName: order.customer.name });
```

Comment:
This should have the customer name in it after "sent to ___ that their"

Reply:
@dt2patel sir, this is handled here https://github.com/hotwax/bopis/pull/212/files#diff-b1791c7a47875ce85f2f1e57ef6c25a9b993660f9bcc840d8f21daac6161b5a0R5
You can also verify it in the screen recording I've added in the issue description.